aspnetcore/tutorials/first-mvc-app/start-mvc/sample/7.0-completed/Controllers/HelloWorld.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Encodings.Web;
+
+namespace MvcMovie.Controllers
```

Comment:
```suggestion
namespace MvcMovie.Controllers;
```

Reply:
I thought of switching to file-scoped namespaces, but the generated code still uses old style, so it seemed better to leave other cases of old style alone so everything would be consistent.

Reply:
We've been using file-scoped for all code we generate or even templated generated when we make substantial changes. It looks much better. The templates should catch up soon, we should lead the way. 

@guardrex, are you using file scoped namespaces or 1990's style?

Reply:
Blazor templates started using them in 6.0; so insofar as I used template code, I started back then. However, I only started converting other examples recently.

I wish that they had gone a different way with line placement to save another vertical line ... https://github.com/dotnet/aspnetcore/issues/44676 ... but they said *'Nope!'* 😢